mods/ra/maps/allies-05a/allies05a-AI.lua

```diff
@@ -243,25 +243,32 @@ ProduceAircraft = function()
 	end
 
 	ussr.Build(SovietAircraftType, function(units)
-		Yaks[#Yaks + 1] = units[1]
+		local yak = units[1]
+		Yaks[#Yaks + 1] = yak
 
 		if #Yaks == 2 then
-			Trigger.OnKilled(units[1], ProduceAircraft)
+			Trigger.OnKilled(yak, ProduceAircraft)
```

Comment:
This will only put the trigger on the second yak, not the first one. I can't imagine that's what you had in mind.